example/src/App.js

```diff
@@ -8,12 +8,11 @@ export default function App() {
   return (
     <div style={{ width: 300, height: 300 }}>
       <InfiniteLoading
-        items={items}
-        itemHeight={40}
         hasMoreItems={false}
+        itemHeight={40}
         loadMoreItems={() => {}}
       >
-        {({ item }) => <div>{item}</div>}
+        {items.map(item => <div key={item}>{item}</div>)}
```

Comment:
[Keys](https://en.reactjs.org/docs/lists-and-keys.html#keys) are meant to help React handle changes to the DOM. Since [elements](https://github.com/bvaughn/react-window/blob/master/src/createListComponent.js#L327) are created by `react-window` (it adds keys on his own) I think this key isn't used at all. What do you think of it ?

Reply:
What does the linter says? I don't think it really matters if you have a key or not but I wouldn't want to add a comment to disable eslint rule everytime I use the lib.

Reply:
If i remove it, i have this warning:
![image](https://user-images.githubusercontent.com/8778672/65283735-319e6500-db0e-11e9-8a3c-f7aa5f71e0b1.png)

I think the user of react-simple-infinite-loading, doesn't know that is not necessary. He only see an array of elements, and each element in an array needs a key.